doc/usage.rst

```diff
@@ -142,6 +142,7 @@ Note: You need to pass the config to the ``fix`` command, in order to make it wo
 
 .. code-block:: console
 
+    export PHP_CS_FIXER_EXPERIMENTAL_PARALLEL_CACHE=1
     php php-cs-fixer.phar list-files --config=.php-cs-fixer.dist.php | xargs -n 10 -P 8 php php-cs-fixer.phar fix --config=.php-cs-fixer.dist.php --path-mode intersection -v
```

Comment:
```suggestion
    PHP_CS_FIXER_EXPERIMENTAL_PARALLEL_CACHE=1 php php-cs-fixer.phar list-files --config=.php-cs-fixer.dist.php | xargs -n 10 -P 8 php php-cs-fixer.phar fix --config=.php-cs-fixer.dist.php --path-mode intersection -v
```
IMHO it shouldn't be exported, but defined only for this particular run, that is supported by external parallelisation. Exporting will keep that value and will use that feature also for regular run (without `xargs`) and I am not sure if it should be like this.

Reply:
```php
var_dump(">>>", getenv('PHP_CS_FIXER_EXPERIMENTAL_PARALLEL_CACHE')); die;
```

your proposal is returning
```php
string(3) ">>>"
bool(false)
```

Reply:
I did not thought about pipes and variable visibility. It should be something like `FOO=bar bash -c 'somecommand someargs | somecommand2'` ([source](https://stackoverflow.com/a/10856348)), in this case variable is available on each level of the pipeline. It makes it a little bit more complicated, but enables feature for single run instead of doing it globally. I won't fight for it, just a suggestion 🙂.

Reply:
i think it makes it overcomplex and I'm ok to have env exported. I assume that one either will want it each time, or never - and if they need sometimes, they will figure out the way.

Reply:
solved in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7131#discussion_r1257916431